packages/truffle-db/src/workspace/test/index.ts

```diff
@@ -129,87 +179,53 @@ mutation AddBytecode($bytes: Bytes!) {
   }
 }`;
 
-it("adds bytecode", async () => {
-  const workspace = new Workspace();
-  const variables = {
-    id: soliditySha3(Migrations.bytecode),
-    bytes: Migrations.bytecode
-  }
-
-  // add bytecode
-  {
-    const result = await graphql(
-      schema, AddBytecode, null, { workspace }, {
-        bytes: variables.bytes
-      }
-    );
-
-    const { data } = result;
-    expect(data).toHaveProperty("bytecodesAdd");
+describe("Bytecode", () => {
+  it("adds bytecode", async () => {
+    const client = new WorkspaceClient();
+    const variables = {
+      id: soliditySha3(Migrations.bytecode),
```

Comment:
Same here, `const expectedId = generateId({ bytes: Migrations.bytecode })`

Note that I'm implicitly standardizing on using objects for **all** IDs

Reply:
I see that! I'll change it for sourceId and bytecodeId.